src/migrations/migrations/migrateNotificationSettingsToV3.ts

```diff
@@ -0,0 +1,21 @@
+import { Migration, MigrationName } from '@/migrations/types';
+import { getAllWalletNotificationSettingsFromStorage, setAllWalletNotificationSettingsToStorage } from '@/notifications/settings/storage';
+
+export function migrateNotificationSettingsToV3(): Migration {
+  return {
+    name: MigrationName.migrateNotificationSettingsToVersion3,
+    async migrate() {
+      const walletSettings = getAllWalletNotificationSettingsFromStorage();
+
+      // reset successfullyFinishedInitialSubscription
+      if (walletSettings.length) {
+        const newSettings = walletSettings.map(wallet => ({
+          ...wallet,
+          successfullyFinishedInitialSubscription: false,
```

Comment:
main change: reset the `successfullyFinishedInitialSubscription` flag so we know to resubscribe using the new endpoint (after it successfully resubscribes, it'll get set back to `true`)